src/index.scss

```diff
@@ -9,6 +10,12 @@ html {
   -moz-osx-font-smoothing: grayscale;
 }
 
+body {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
```

Comment:
Why is HTML and body height 100% if the element nested in them has a height of 100vh?